src/komponenter/utvandret/oppgave-opprettet.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { Feilmelding, Undertittel } from 'nav-frontend-typografi';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import Veilederpanel from 'nav-frontend-veilederpanel';
+import Alertstripe from 'nav-frontend-alertstriper';
+import virkedager from '@alheimsins/virkedager';
+import { Kontaktinformasjon } from '../oppholdstillatelse';
+import EksternLenke from '../ekstern-lenke/ekstern-lenke';
+import { getHeaders, MED_CREDENTIALS } from '../../ducks/api';
+import prettyPrintDato from '../../utils/pretty-print-dato';
+import veilederSvg from '../oppholdstillatelse/veileder-mann.svg';
+import { uniLogger } from '../../metrikker/uni-logger';
```

Comment:
Skal vi vurdere å gå bort fra uniLogger og heller bare kjøre Amplitude på denne typen logging av aktivitet?
Unilogger ble innført da vi testet Amplitude og ville sikre oss data i begge systemer. Nå har Amplitude kommet for å bli, så det er kanskje ikke verdifullt med disse metrikkene i Grafana også?

Reply:
Enig!